nodejs/pulumi/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "pulumi",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.mjs",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
@jkodroff I know that the default license type is `ISC` but could you change to `Apache-2.0`? The whole `dagger/examples` repo is under the `Apache-2.0` license and keeping this consistent will mean that we don't need to change https://github.com/dagger/examples/blob/main/LICENSE

Reply:
Fixed.